src/components/views/donate/NiceBanner.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Image from 'next/image';
+import styled from 'styled-components';
+import {
+	brandColors,
+	GLink,
+	IconExternalLink,
+	H4,
+	Lead,
+} from '@giveth/ui-design-system';
+import { mediaQueries } from '@/lib/constants/constants';
+import { IProjectBySlug } from '@/apollo/types/types';
+import links from '@/lib/constants/links';
+import Spark from '/public/images/spark.svg';
+
+const NiceBanner = (props: IProjectBySlug) => {
+	const { project } = props;
+	// Only show this on the Giveth project}
+	if (project.id !== '1') return null;
+	return (
+		<Wrapper>
+			<Container>
+				<Content>
+					<Title>
+						Get $nice <Image src={Spark} alt='spark' />
```

Comment:
we have IconSpark in the ui-design-system and I think it's better to use it instead